index.ftd

```diff
@@ -156,241 +235,214 @@ figma-json: $color-display-one.figma-json
 
 
 
+-- component title-with-link:
+caption title:
+optional ftd.image-src icon:
+optional string link-text:
+optional string link:
+boolean $mouseover: false
 
 
+-- ftd.column:
+width: fill-container
+spacing.fixed.px: 6
 
+-- ftd.text: $title-with-link.title
+color: #212738
+role: $inherited.types.button-small
+style: bold
+
+-- ftd.row:
+align-content: center
+spacing.fixed.px: 9
+link: $title-with-link.link
+color: $inherited.colors.info.text
+$on-mouse-enter$: $ftd.set-bool($a = $title-with-link.mouseover, v = true)
+$on-mouse-leave$: $ftd.set-bool($a = $title-with-link.mouseover, v = false)
+
+-- ftd.image: 
+if: { title-with-link.icon != NULL }
+src: $title-with-link.icon
+
+-- ftd.text: $title-with-link.link-text
+if: { title-with-link.link-text != NULL }
+role: $inherited.types.button-small
+color: #006EB3
+style: bold
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: title-with-link
 
 
 
 
 
 
--- component color-display-data:
-ftd.color-scheme color:
 
--- ftd.column:
-width.fixed.percent: 40
-padding-left.px: 24
 
--- colors:
-color-name: `$inherited.colors.background.base`
-bg-color: $color-display-data.color.background.base
 
--- colors:
-color-name: `$inherited.colors.background.step-1`
-bg-color: $color-display-data.color.background.step-1
 
--- colors:
-color-name: `$inherited.colors.background.step-2`
-bg-color: $color-display-data.color.background.step-2
+-- component markdown:
+body body:
+optional boolean collapsed:
+optional caption title:
+optional boolean two_columns:
 
--- colors:
-color-name: `$inherited.colors.background.overlay`
-bg-color: $color-display-data.color.background.overlay
+-- ftd.text: 
+text: $markdown.body
+role: $inherited.types.copy-regular
+color: #212738
 
--- colors:
-color-name: `$inherited.colors.background.code`
-bg-color: $color-display-data.color.background.code
+-- ftd.text.classes:
```

Comment:
This can be simplified since classes accepts a string list, we can use the shorthand string syntax
 ```suggestion
classes: markdown
```